au.yaml

```diff
@@ -162,10 +162,6 @@ months:
     regions: [au_tas_north]
     wday: 1
     week: 1
-  - name: Melbourne Cup Day
-    regions: [au_vic_melbourne, au_vic]
```

Comment:
@chrisops I'm pretty sure the only change you needed to make was that the regions here was "au_vic_melbourne"